src/Data/TypeLevel/Num/Ops.hs

```diff
@@ -386,7 +386,7 @@ infixr 8 ^
 -- @log_base_b x = e@
 --  Note it is not relational (i.e. cannot be used to express exponentiation)
 class (Pos b, b :>=: D2, Pos x, Nat e) =>  LogBase b x e  | b x -> e 
-instance  LogBaseF b x e f => LogBase b x e
+instance  (Pos b, b :>=: D2, Pos x, Nat e, LogBaseF b x e f) => LogBase b x e
```

Comment:
I've read the release notes and your fix looks fine. It's a pity that the code becaomes so verbose though, so I would favor disabling the warning instead due to our abuse of typeclasses I think it would be justified.